src/app/dts/template-editor/template-editor.component.ts

```diff
@@ -3,53 +3,39 @@ import {FormBuilder, FormGroup, Validators} from '@angular/forms';
 import {DtsService} from '../dts.service';
 import {Template} from '../template.types';
 import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material/dialog';
-import {CKEditor4} from 'ckeditor4-angular';
 import {debounceTime, switchMap} from 'rxjs/operators';
 import {UnsubscribeOnDestroyAdapter} from '../unsubscribe-on-destroy-adapter';
 
+declare var CKEDITOR: any;
```

Comment:
You could export this declaration so that your unit test could import it and pass:
```export declare var CKEDITOR: any;```
A better option would be to get the Typescript types for CKEDITOR using DefinitelyTyped: https://github.com/DefinitelyTyped/DefinitelyTyped

Reply:
Will comment out test, investigate the two proposed options and address this latter